Contributing.md

```diff
@@ -82,4 +82,38 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
```

Comment:
We should add this to the vocabulary section. Also the TC process likely needs to be updated to involved the captains, since the wording here of what they do seems to out weight committers.

Reply:
I have made amendments, please have a look!

Reply:
Lgtm